xonsh/xoreutils/_which.py

```diff
@@ -345,11 +340,11 @@ def main(argv):
     for arg in args:
         #print "debug: search for %r" % arg
         nmatches = 0
-        for match in whichgen(arg, path=altpath, verbose=verbose, exts=exts):
+        for absName, fromWhere in whichgen(arg, path=altpath, verbose=verbose, exts=exts):
```

Comment:
picking nits, but these should be PEP8 variable names (snakes instead of camels)